priprava/instalace-nastroju/overeni.md

```diff
@@ -13,6 +13,24 @@ Postupujte dle následujicích kroků.
 
    Tento příkaz spustí dotazník, který se vás zeptá na váš e-mail a jméno. E-mail zadejte ten, který jste použili při vytváření účtu na GitHubu.
 
-1. Pokud program záhlásí, že všechno proběhlo v pořádku, slavte úspěch. Pokud se cestou cokoliv pokazilo, pomohou vám naši lektoři a kouči.
+   ::fig[ukázka běhu ověření]{src=assets/overeni.gif}
+
+1. Pokud program zahlásí, že všechno proběhlo v pořádku, slavte úspěch. Pokud se cestou cokoliv pokazilo, napište do kanálu `#04-otazky`` na Slacku, lektoři a koučové s vámi problém vyřeší.
```

Comment:
A nejmenuje se ten, nebude se ten kanál jmenovat jinak? Většinou bývají s podtržítky. `#04-otazky` vs `#04_otazky`

Reply:
Ptal jsem se Nikol, zda bude současný Slack nebo jiný a jak se kanál bude jmenovat. Až to bude potvrzené, dám tam i odkaz. Každopádně teď je tam podtržítko, opravím. A to jsem se na to díval asi 3×, zda je tam podtržítko nebo pomlčka…

Reply:
Co mi popisovala Niky, tak už je založený nový workspace na Slacku, který bude platit pro podzimní a všechny budoucí webové akademie.

Po doběhnutí každé akademie se všechny příslušné kanály, pokud jsem to dobře pochopil, přemění na private. V názvech kanálů by pak na konci měly být koncovky s identifikátorem konkrétního běhu, aby nedošlo ke kolizi při zakládání nových public pro nový běh. Jestli i ty public budou mít koncovku nevím. Jestli to říkám správně taky nevím. Každopádně jsem si celkem jistý, že tady nemůže být odkaz, pokud si to tedy nechceme důsledně hlídat a před každým během ho měnit, protože by jinak časem odkaz vedl na kanál staršího běhu.